.github/workflows/build-website.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           ls website/build
 
       - name: Upload Build Artifact
-        uses: actions/upload-pages-artifact@v3
+        uses: actions/upload-pages-artifact@v4
```

Comment:
Hmmm this is weird - dependabot isn't using hashes as it should. is there a way to force it to do that?

Reply:
I don't think I've ever seen dependabot update actions based on hashes.

Reply:
It does!! 

See this pr: 
https://github.com/stravalib/stravalib/pull/662/files 

Let's merge this, but I'll add a hash, and we will see what it does in the future. It can magically do this, but i don't understand how the magic works. 🙃